pkg/storage/mysql/mysql_test.go

```diff
@@ -1,21 +1,112 @@
-package mysql_test
+package mysql
 
 import (
+	"context"
 	"testing"
+	"time"
 
-	"github.com/openfga/openfga/pkg/storage/common"
-	"github.com/openfga/openfga/pkg/storage/mysql"
+	sq "github.com/Masterminds/squirrel"
+	"github.com/openfga/openfga/pkg/storage"
+	"github.com/openfga/openfga/pkg/storage/sqlcommon"
 	"github.com/openfga/openfga/pkg/storage/test"
 	storagefixtures "github.com/openfga/openfga/pkg/testfixtures/storage"
+	"github.com/openfga/openfga/pkg/tuple"
 	"github.com/stretchr/testify/require"
+	openfgav1 "go.buf.build/openfga/go/openfga/api/openfga/v1"
 )
 
 func TestMySQLDatastore(t *testing.T) {
 	testDatastore := storagefixtures.RunDatastoreTestContainer(t, "mysql")
 
 	uri := testDatastore.GetConnectionURI()
-	ds, err := mysql.New(uri, common.NewConfig())
+	ds, err := New(uri, sqlcommon.NewConfig())
 	require.NoError(t, err)
 	defer ds.Close()
 	test.RunAllTests(t, ds)
 }
+
+// TestReadEnsureNoOrder asserts that the read response is not ordered by ulid
+func TestReadEnsureNoOrder(t *testing.T) {
+	testDatastore := storagefixtures.RunDatastoreTestContainer(t, "mysql")
+
+	uri := testDatastore.GetConnectionURI()
+	ds, err := New(uri, sqlcommon.NewConfig())
+	require.NoError(t, err)
+	defer ds.Close()
+
+	ctx := context.Background()
+	store := "store"
+	objectType := "doc"
+	objectID1 := "object_id_1"
+	relation := "relation"
+	user1 := "user:user_1"
+	firstTuple := tuple.NewTupleKey(objectType+":"+objectID1, relation, user1)
+
+	objectID2 := "object_id_2"
+	user2 := "user:user_2"
+	secondTuple := tuple.NewTupleKey(objectType+":"+objectID2, relation, user2)
+
+	err = sqlcommon.Write(ctx, sqlcommon.NewDBInfo(ds.db, ds.stbl, sq.Expr("NOW()")), store, []*openfgav1.TupleKey{},
```

Comment:
minor nitpick, not critical

Would you mind formatting these lines so they wrap. They are challenging to read.

```go
err = sqlcommon.Write(
  ctx,
  sqlcommon.NewDBInfo(...),
  store,
  []*openfgav1.TupleKey{},
  []*oepnfgav1.TupleKey{firstTuple},
  time.Now(),
)
```

It'd be nice if we could enforce a golint rule for this. I'll look into that.

Reply:
done